packages/rfw/test/runtime_test.dart

```diff
@@ -1088,4 +1088,286 @@ void main() {
     data.update('c', 'test');
     expect(log, <String>['leaf: 2', 'root: {a: [2, 3], b: [q, r]}', 'root: {a: [2, 3], b: [q, r], c: test}']);
   });
+
+  group('Widget Builders', () {
+    late DynamicContent data;
+    late List<RfwEvent> dispatchedEvents;
```

Comment:
please avoid groups and instead put all the shared state into each test, that way we can much more easily change individual tests, refactor them, move them to other files, etc.

Reply:
Great call. Done!